docs/bokeh/source/docs/user_guide/basic/data.rst

```diff
@@ -226,15 +226,12 @@ For example: If a ``DataFrame`` has the columns ``'year'`` and ``'mpg'``,
 passing ``df.groupby('year')`` to a ColumnDataSource will result in columns such
 as ``'mpg_mean'``.
 
-.. note::
-    Adapting ``GroupBy`` objects requires pandas version 0.20.0 or above.
-
 .. _ug_basic_data_cds_streaming:
 
 Appending data to a ColumnDataSource
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-|ColumnDataSource| streaming is an efficient way to append new data to a
+ColumnDataSource streaming is an efficient way to append new data to a
```

Comment:
The `|` fences create a Sphinx alias for a link to the refguide:

<img width="353" alt="Screenshot 2023-05-30 at 21 32 58" src="https://github.com/bokeh/bokeh/assets/1078448/0f25f5b4-eecc-4f18-8dc9-9482dd5cd771">

These should be retained.